CONTRIBUTING.md

```diff
@@ -60,6 +66,126 @@ cd api-bench
 npm run setup
 ```
 
+## Commit Message Guidelines
+
+**This specification is inspired by Angular commit messages guidelines.*
+
+These are the rules for how Git commit messages for api-bench should be formatted.
+This format leads to easier to read commit history.
+
+Each commit message consists of a **header**, a **body**, and a **footer**.
+
+```md
+<header>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+The `header` is mandatory and should conform to the
+[Commit Message Header](#commit-message-header) format.
+
+The `body` is mandatory for all commits except for those of type "docs"
+or in cases when it's clear from the header summary. They should conform
+to the [Commit Message Body](#commit-message-body) format.
+
+The `footer` is optional. The [Commit Message Footer](#commit-message-footer)
+format describes what the footer is used for and the structure it must have.
+
+### Commit Message Header
+
+Headers must adhere to the following format:
+
+```md
+<type>(<scope>): <short summary>
+
+Example:
+build(website): bump follow-redirects from 1.15.4 to 1.15.6
+```
+
+#### Type
+
+The list of supported types:
+
+- `build`: Changes that affect the build system or external dependencies
+- `ci`: Changes to CI configuration files and scripts
+- `docs`: Documentation changes
+- `feature`: A new feature
+- `fix`: A bug fix
+- `perf`: A code change that improves performance
+- `refactor`: A code change that neither fixes a bug nor adds a feature
+- `test`: Adding missing tests or correcting existing tests
+
+#### Scope
+
+The scope should be the name of the affected part of the project.
+The list of supported scopes:
+
+- `framework`
+- `website`
+- `history-microservice`
+- `history-website`
+
+#### Summary
+
+The summary field should provide a succinct description of the change.
+
+- use the imperative, present tense
+- don't capitalise the first letter
+- no dot (.) at the end
+
+```js
```

Comment:
you can just remove the `js` here - nothing to highlight in pure text :)

Reply:
I added language not because of highlighting, but because without it markdownlint throws this error:
`MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "```"]`

Reply:
then md may be less confusing

Reply:
maybe, but I wanted to separate the comments from the actual text and chose js as one of the most popular langs. md just paints everything gray while js colors comments and text differently (both in README and in the browser). But if you want md, so be it.